centreon/packages/ui/src/Graph/BasicComponents/Axes/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { Axis } from '@visx/visx';
+import { ScaleLinear } from 'd3-scale';
+import { isNil } from 'ramda';
+
+import { useLocaleDateTimeFormat } from '@centreon/ui';
+
+import { getXAxisTickFormat } from '../../helpers';
+import { GraphInterval } from '../../models';
+import { getUnits } from '../../timeSeries';
+
+import UnitLabel from './UnitLabel';
+import { Data } from './models';
+import useAxisY from './useAxisY';
+
+interface Props {
+  data: Data;
+  graphInterval: GraphInterval;
+  height: number;
+  leftScale: ScaleLinear<number, number>;
+  rightScale: ScaleLinear<number, number>;
+  width: number;
+  xScale: ScaleLinear<number, number>;
+}
+
+const Axes = ({
+  height,
+  width,
+  data,
+  rightScale,
+  leftScale,
+  xScale,
+  graphInterval
+}: Props): JSX.Element => {
+  const { format } = useLocaleDateTimeFormat();
+  const { lines } = data;
+
+  const { axisLeft, axisRight } = useAxisY({ data, graphHeight: height });
+
+  const [firstUnit, secondUnit, thirdUnit] = getUnits(lines);
+
+  const xTickCount = Math.ceil(width / 82);
+
+  const tickFormat =
+    data?.axisX?.xAxisTickFormat ?? getXAxisTickFormat(graphInterval);
+
+  const formatAxisTick = (tick): string =>
+    format({ date: new Date(tick), formatString: tickFormat });
```

Comment:
- tick marks for every hour or day, if not enough space to show them all with labels then have them as minor ticks (without label) :  the ticks are displayed depend on the graph width , if there is enough space ticks will be displayed for each hour , but show a fix numbers of  ticks without information we will lose this feature of responsivity ,  maybe that should be discussed with PO , cause that will change the current behavior of the graph 🤔 
- for time and date , should have just confirmation because that will change the current graph 🤔 
- numTick : it s responsible of showing the ticks , and because we have a responsive graph (width and hieght changes depend on space ) we display the ticks depends on space 
- for numbers like width/82 or 30 , that for changing the number of num ticks , the default value is 10 , but why exactly 82 or 30 idk , but there a the perfect numbers to divise on (this funstionnality is already developed  ) for my side i tested and it works perfectly :police_car: :police_car:  👀